azurerm/resource_arm_relay_namespace.go

```diff
@@ -94,13 +107,19 @@ func resourceArmRelayNamespace() *schema.Resource {
 func resourceArmRelayNamespaceCreateUpdate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*ArmClient).relayNamespacesClient
 	ctx := meta.(*ArmClient).StopContext
+
+	sku := expandRelayNamespaceSku(d)
```

Comment:
As above lets pull out the logic as its so small

Reply:
Yup... pulled logic out and put it at the top of  `resourceArmRelayNamespaceCreateUpdate`